kodilla-patterns/src/main/java/com/kodilla/patterns/factory/tasks/TaskFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.patterns.factory.tasks;
+
+public final class TaskFactory {
+    public static final String SHOPPING = "SHOPPING";
+    public static final String PAINTING = "PAINTING";
+    public static final String DRIVING = "DRIVING";
+
+    public Task executeTask(final String taskClass) {
+        switch (taskClass) {
+            case SHOPPING:
+                return new ShoppingTask("Shopping in LA.","shoes", 6 );
```

Comment:
bez sensu. A jak chce teraz miec Shopping in Chicagio jedna sztuke spodni? To juz moje factory mi nie zadziala.

Reply:
Nie wiem, czy do końca Cię zrozumiałem. Usunąłem miasto, ponieważ generalnie nie ma o nim mowy.